mangaki/mangaki/models.py

```diff
@@ -7,9 +7,13 @@
 from django.contrib.contenttypes.models import ContentType
 from django.core.urlresolvers import reverse
 
+import os.path
+
 from mangaki.discourse import get_discourse_data
 from mangaki.choices import ORIGIN_CHOICES, TYPE_CHOICES, TOP_CATEGORY_CHOICES
 from mangaki.utils.ranking import TOP_MIN_RATINGS, RANDOM_MIN_RATINGS, RANDOM_MAX_DISLIKES, RANDOM_RATIO
+from mangaki.settings import MEDIA_URL, STATIC_URL, MEDIA_ROOT
```

Comment:
Plutôt `from django.conf import settings` puis `settings.MEDIA_URL` etc.; cf https://docs.djangoproject.com/en/1.9/topics/settings/#using-settings-in-python-code